sdks/java/core/src/main/java/org/apache/beam/sdk/io/LocalFileSystem.java

```diff
@@ -176,8 +177,15 @@ protected String getScheme() {
   }
 
   private MatchResult matchOne(String spec) throws IOException {
-    File file = Paths.get(spec).toFile();
+    if (spec.toLowerCase().startsWith("file:")) {
+      spec = spec.substring("file:".length());
+    }
 
+    if (SystemUtils.IS_OS_WINDOWS) {
+      spec = spec.replaceAll("^/+", "");
```

Comment:
This won't work for filenames with / in them like c://blah//foo.txt?

If this is to get rid of file:/// or file:/ maybe we should just handle those cases directly instead of checking to see if this is windows and removing the prefix?